node/src/signable.d.ts

```diff
@@ -9,12 +9,13 @@
  */
 export interface Signable {
     /**
-     * Get the serialized commit status request message.
+     * Get the serialized bytes of the signable object.
+     * Serialized bytes can be used to recreate the object using methods on Gateway.
```

Comment:
It would be good to use {@link Gateway} so users can more easily navigate to the related documentation.